### PR TITLE
Adds some docs and fixes minor nits/style issues

### PR DESCRIPTION
**Pull request by ypc-faros**
# Description

Adds comment pointing to PR explaining the double loading of Hasura metadata.
Also fixes some minor style issues.

# Checklist
- [X] Have you checked to there aren't other open Pull Requests for the same update/change?
- [X] Have you lint your code locally before submission?
- [X] Have you added an explanation of what your changes do and why you'd like us to include them?

### CODE REVIEW

File: init/src/airbyte/init.ts [hunk 1]
```diff
@@ -271,8 +271,13 @@ async function main(): Promise<void> {
   program.parse();
   const options = program.opts();
 
-  if (options.airbyteApiCallsConcurrency !== undefined && options.airbyteApiCallsConcurrency <= 0) {
-    throw new InvalidArgumentError('airbyte-api-calls-concurrency must be a positive integer');
+  if (
+    options.airbyteApiCallsConcurrency !== undefined &&
```
Comment (tovbinm) [inline init/src/airbyte/init.ts:275]: `options.airbyteApiCallsConcurrency && options.airbyteApiCallsConcurrency <= 0`

**Review by tovbinm — APPROVED** [review]

File: init/src/airbyte/init.ts (hunk 1 above)
Comment (ypc-faros) [reply init/src/airbyte/init.ts:275]: I think if I remove the `!== undefined` it won't fail if they specify `--airbyte-api-calls-concurrency 0`

Comment (tovbinm) [reply init/src/airbyte/init.ts:275]: ok, keep it as you have it. but what about `null`?

Comment (tovbinm) [reply init/src/airbyte/init.ts:275]: perhaps do `options?.airbyteApiCallsConcurrency <= 0`

Comment (ypc-faros) [reply init/src/airbyte/init.ts]: Nice! That works

**Pull request merged**